io.openems.edge.controller.ess.cycle/src/io/openems/edge/controller/ess/cycle/Config.java

```diff
@@ -17,29 +17,35 @@
 	@AttributeDefinition(name = "Is enabled?", description = "Is this Component enabled?")
 	boolean enabled() default true;
 
+	@AttributeDefinition(name = "Mode", description = "Set the type of mode.")
+	Mode mode() default Mode.MANUAL_ON;
+
+	@AttributeDefinition(name = "Hybrid-ESS Mode", description = "For Hybrid-ESS (ESS with attached DC-side PV system): apply target power to AC or DC side of inverter?")
+	HybridEssMode hybridEssMode() default HybridEssMode.TARGET_AC;
+
 	@AttributeDefinition(name = "Ess-ID", description = "ID of Ess device.")
 	String ess_id() default "ess0";
 
 	@AttributeDefinition(name = "CycleOrder [Charge/Discharge]", description = "Charge/discharge CycleOrder for Operation")
 	CycleOrder cycleOrder() default CycleOrder.START_WITH_DISCHARGE;
 
-	@AttributeDefinition(name = " Standby Time in Minute", description = "Wait Between Charge and Discharge (Just Integer in time).")
-	int standbyTime() default 1;
+	@AttributeDefinition(name = " Standby Time in Minute", description = "Wait Between Charge and Discharge (Just Integer in time minute).")
+	int standbyTime() default 30;
 
-	@AttributeDefinition(name = " Start Time in Hour [YYYY-MM-DD HH:MM]", description = "Start Charge/Discharge Hour (2021-09-02 13:35).")
-	String startTime() default "2021-09-07 13:35";
+	@AttributeDefinition(name = "Start Time in Hour [YYYY-MM-DD HH:MM]", description = "Start Charge/Discharge Hour (2023-03-23 15:35).")
+	String startTime() default "2023-03-23 15:35";
```

Comment:
I would not change configurations like the date to a more recent date. I would pick some generic date like '2000-01-01 12:00'. If you want to provide a configuration example i would do it via documentation

Reply:
I wanted to write it directly here, becuase it was simple to configure it and no body checks the docs. Or we might just let this " Charge/Discharge Hour (2023-03-23 15:35) " in config as an explanation.